aiosmtpd/smtp.py

```diff
@@ -207,6 +263,35 @@ def smtp_QUIT(self, arg):
             self.transport.close()
 
     @asyncio.coroutine
+    def smtp_STARTTLS(self, arg):
+        log.info('===> STARTTLS')
+        if arg:
+            yield from self.push('501 Syntax: STARTTLS')
+            return
+        if (not self.tls_context) or (not _has_ssl):
+            yield from self.push(
+                '454 TLS not available due to temporary reason'
+            )
+            return
+        yield from self.push('220 Ready to start TLS')
+        # create SSL layer
+        self._tls_protocol = sslproto.SSLProtocol(
+            self.loop,
+            self,
+            self.tls_context,
+            None,
+            server_side=True
+        )
+        # reconfigure transport layer
+        socket_transport = self.transport
```

Comment:
Indent paren 4 spaces.

Please use complete sentences in the comments (i.e. start with a capital and end in a period).  Also for subsequent comments in this chunk.